raft/src/test/java/org/apache/kafka/raft/RequestManagerTest.java

```diff
@@ -256,25 +398,25 @@ public void testAnyInflightRequestWithAnyRequest() {
             random
         );
 
-        assertFalse(cache.hasAnyInflightRequest(time.milliseconds()));
```

Comment:
Do you want to parametrize this test and do it over all of the ApiKeys?

Reply:
You can think of tests as a form of documentation. All of the KRaft RPCs that are not FETCH and FETCH_SNAPSHOT behave in a certain way while FETCH and FETCH_SNAPSHOT behave in a different way.